install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/usr.sbin.apache2

```diff
@@ -1,4 +1,4 @@
-# Last Modified: Wed Oct 29 08:16:32 2014
+# Last Modified: Wed Feb 10 16:27:32 2021
```

Comment:
Clearly not the most well-maintained timestamp. ;-) Maybe we should just ditch it?